app/assets/javascripts/components/button_start.jsx

```diff
@@ -0,0 +1,14 @@
+function ButtonStart(props) {
+
+    return (
+      <td>
+        <button 
+          onClick={ props.changeFightProperty }
+          disabled={ props.btnClick } 
+          className="btn btn-start" 
+        >
+        STRIKE
```

Comment:
For the future references text should not be hardcoded  but localize through I18n, but this one not in the scope of this review.